eng/packages/http-client-csharp/emitter/test/Unit/options.test.ts

```diff
@@ -0,0 +1,32 @@
+import { Program } from "@typespec/compiler";
+import { beforeEach, describe, it } from "vitest";
+import { createEmitterContext, createEmitterTestHost, typeSpecCompile } from "./test-util.js";
+import { AzureEmitterOptions } from "../../src/options.js";
+import { $onEmit } from "../../src/emitter.js";
+import { strictEqual } from "assert";
+
+describe("Configuration tests", async () => {
+    let program: Program;
+  
+    beforeEach(async () => {
+      const runner = await createEmitterTestHost();
+      program = await typeSpecCompile(
+        `
+          `,
+        runner,
+      );
+    });
+  
+    it("Diagnostic is logged when model-namespace is set without namespace", async () => {
```

Comment:
maybe add a test that verifies the diag does not exist if both configs are set and neither config is set.